inc/classes/admin/class-options.php

```diff
@@ -1,8 +1,6 @@
 <?php
 namespace WP_Rocket\Admin;
 
-defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
```

Comment:
This should not be removed.
A direct access to the file will print a fatal error leading to a path disclosure:
```
Fatal error: Class 'WP_Rocket\Admin\Abstract_Options' not found in /abs/path/to/inc/classes/admin/class-options.php on line 10
```